lib/codebreaker/validate_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# validation method for codebreaker
+module Codebreaker
+  CODE_SIZE = 4
```

Comment:
U have already defined `CODE_SIZE` in `SecretCode`. U have to define constant only once through the project, so if u need to change it, u'll change it only in one place.